WordPressAuthenticator.podspec

```
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'WordPressAuthenticator'
-  s.version       = '3.2.0'
+  s.version       = '3.2.1-beta1'
```

[nit] I think we can bump the minor version instead of the micro version - wdyt?
```suggestion
  s.version       = '3.3.0-beta.1'
```